js/util/wizard_agency_search.js

```diff
@@ -124,6 +124,9 @@ function searchMatchingAgency(query, flatList, debug = false) {
     };
   });
 
+  // Common words that are also abbreviations which we don't want to match.
```

Comment:
```suggestion
  // Some agencies have "abbreviations" that are instead common words.
  // We don't match them because it would too heavily weigh the results in
  // favor of showing just agency matches instead of model results.
```